pkgs/development/python-modules/warp-lang/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, cudatoolkit
+}:
+
+buildPythonPackage rec {
+  pname = "warp-lang";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-htcUS/qPqL2eyT7c+rnE/SQPR7mAVwf/F2OlDtJVMs8=";
+  };
+
+  propagatedBuildInputs = [ cudatoolkit numpy ];
+  pythonImportsCheck = [ "warp" ];
+
+  # Needs CUDA device at runtime to run the tests
+  doCheck = false;
```

Comment:
I tried to run the tests for warp, but during the nix build it  yields:

```
running build_ext
error: libstdc++.so.6: cannot open shared object file: No such file or directory
/build/warp-lang-0.2.0/warp/types.py:13: UserWarning: The NumPy module was reloaded (imported a second time). This can in some cases result in small but subtle issues and is discouraged.
  import numpy as np
error: builder for '/nix/store/zqqh4vg2x6wfag9ink4vhjhpwzzpxn6w-python3.9-warp-lang-0.2.0.drv' failed with exit code 1
```

Reply:
Have you tried adding `stdenv.cc.cc.lib` into your checkInputs to see if it finds the lib?